### PR TITLE
Sidebar restructure

### PR DESCRIPTION
**Pull request by kimburgess**
- Restructure of sidebar metadata to use Docusaurus autgenerated nav.
- Minor neaten up of repository metadata.

_Note:_ `placeos/docs-site` branch reference to be returned to default following merge of https://github.com/PlaceOS/docs-site/pull/27

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -10,23 +14,25 @@ It provides tools to design, build and iterate integrated digital experiences fo
...
+The documentation for PlaceOS is split into 4 categories.
 
-See the [overview](../overview) for high-level coverage of topic areas.
+See the [overview](overview/drivers) for high-level coverage of topic areas.
 
-Try a [tutorial](../tutorial/) to learn in more detail about a process or feature.
+Try a [tutorial](tutorial/write-a-driver) to learn in more detail about a process or feature.
 
-Try a [how-to](../how-to/) for docs to help you with a specific task.
+Read a [how-to guide](how-to/configure-saml) for docs to help you with a specific task.
```
Comment (mghill) [inline index.md:32]: Why do these link to specific documents rather than the section indexes?

**Review by mghill — COMMENTED** [review]
Not sure why we lost the category index pages? I thought there was a solution for having a page for a category. Otherwise it all tracks

**Comment by kimburgess** [comment]
Those pages were dropped as they do not provide any useful information in the current form. They were filler / placeholder items. With the move to all doc categories reachable from the one page, the immediate need for that category landing is lifted.

A category index is something worth having, but we can auto-generate this based on content available within the category. This can also be introduced when sidebar navigation allows for linking to these from the category group (https://github.com/facebook/docusaurus/issues/2643).

**Comment by mghill** [comment]
Valid, happy to revisit that when the functionality is in. In that case, all good with me. Linking to specific ones still feels weird, but I guess the other option is just having them with no link and I can't decide if that's weirder.

**Comment by kimburgess** [comment]
Yeah, that's definitely a temporary kludge.

**Pull request merged**